src/cmd/append.go

```diff
@@ -19,6 +19,18 @@ type appendConfig struct {
 var appendCommand = &cobra.Command{
 	Use:   "append <branch>",
 	Short: "Creates a new feature branch as a child of the current branch",
+	Long: `Creates a new feature branch as a child of the current branch.
+
+Syncs the current branch,
+forks a new feature branch with the given name off the current branch,
+makes the new branch a child of the current branch,
+pushes the new feature branch to the remote repository,
+and brings over all uncommitted changes to the new feature branch.
+
+Additionally, when there is a remote upstream,
+the main branch is synced with its upstream counterpart.
+This can be disabled by toggling the "hack-push-flag" configuration:
+$ git town hack-push-flag false`,
```

Comment:
Reading this it appears the hack-push-flag is related to syncing the main branch when it actually related to the `pushes the new feature branch to the remote repository` on line 27.

This is also duplicated on the hack / prepend commands. Can we DRY that up with a variable so any changes to it would only need to happen in one place.